main.js

```diff
@@ -0,0 +1,101 @@
+var APIkey = "9acb5f4800d2541ee3827f9f17a00dfe";
+
+$('.search').on('click', function () {
+  var query = $('#search-query').val();
+  fetchToday(query);
+  fetchForecast(query);
+});
+
+var fetchToday = function (query) {
+  $.ajax({
+    method: "GET",
+    url: "https://api.openweathermap.org/data/2.5/weather?q=" + query + "&appid=" + APIkey + "&units=imperial",
+    dataType: "json",
+    success: function(data) {
+      findToday(data)
+    },
+    error: function(jqXHR, textStatus, errorThrown) {
+      alert("We could not find weather data for " + query);
+      console.log(textStatus);
+    }
+  });
+};
+
+var findToday = function (data) {
+
+  var iconID = data.weather[0].icon;
+  var iconURL = "http://openweathermap.org/img/wn/" + iconID + "@2x.png"
+
+
+  var today = {
+    temp: Math.round(data.main.temp),
+    name: data.name,
+    iconURL: iconURL,
+    description: data.weather[0].description
+  };
+
+  renderToday(today);
+};
+
+var renderToday = function (today) {
```

Comment:
I might prefer to call the parameter `data` but this works just as well

Reply:
Thank you for grading this!

What sorts of things should I be working on for readability and efficiency?

There are a few things I expanded a bit to make them a bit more clear, but that may have also made things less efficient. Appreciate your feedback!

Reply:
I probably could have rendered both today's forecast and the 5 day forecast together. Is it generally preferable to call parameters data if they are pulling info from JSON?

Reply:
Yes, data makes sense as a parameter name for anything coming back from an API. 

For efficiency, you have `fetchToday`, `findToday` and `renderToday` - fetch and find could probably be combined, same with `renderForecast`.